cmd/subctl/generic_cloud.go

```diff
@@ -0,0 +1,57 @@
+/*
+SPDX-License-Identifier: Apache-2.0
+
+Copyright Contributors to the Submariner project.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package subctl
+
+import (
+	"github.com/spf13/cobra"
+	"github.com/submariner-io/submariner-operator/internal/cli"
+	"github.com/submariner-io/submariner-operator/internal/exit"
+	"github.com/submariner-io/submariner-operator/pkg/cloud/cleanup"
+	"github.com/submariner-io/submariner-operator/pkg/cloud/prepare"
+)
+
+var (
+	gateways int
```

Comment:
Let's encapsulate this in a struct.

```suggestion
         genericCloudConfig struct {
		gateways int
	}
```

Reply:
Its juts 1 var and not reused by `cleanup` so left it as is.

Reply:
The point is to isolate the config/options for each command so there's no hidden, accidental sharing with other commands since they all reside in the same package. We've used anonymous structs for other commands, eg `export`, `unexport`,  so let's be consistent and do the same here.